src/clusterfuzz/_internal/cron/project_setup.py

```diff
@@ -864,6 +885,17 @@ def _sync_job(self, project, info, corpus_bucket_name, quarantine_bucket_name,
       file_github_issue = info.get('file_github_issue', False)
       job.environment_string += f'FILE_GITHUB_ISSUE = {file_github_issue}\n'
 
+      # Android creates device-specific queues during project setup.
+      queue_id = info.get('queue_id', False)
+      if queue_id:
+        if 'android' not in job.templates:
+          job.templates.append('android')
+        if template.architecture == 'arm':
+          job.platform = queue_id.split('-')[1].upper()
+        else:
+          job.platform = ANDROID_CUTTLEFISH
```

Comment:
Can  you please add a comment for why these cases are different?

Reply:
This is no longer necessary as the `job.platform` is being constructed from the variables passed in